gitops/components/messaging/rabbitmq/single-instance/secrets.yaml

```diff
@@ -0,0 +1,9 @@
+kind: Secret
+apiVersion: v1
+metadata:
+  name: rabbitmq-secret
+data:
+  cookie: S1Y4MnhjNnA3MXJpU0ppZw==
```

Comment:
We can't commit real secrets, be sure these are faked and call that out in the readme. shortly in the future we should be able to use Hashicorp Vault for our secrets

Reply:
sure